### ISSUE TITLE
Add message handling for JavaScriptEvalOutput (via Date)

### ISSUE THREAD

**Issue by linclark** [bgrins/console-output-experiment#6]
<img width="236" alt="obj_renderer_date" src="https://cloud.githubusercontent.com/assets/105464/13023286/5e5eae26-d19c-11e5-87a3-896f119fed34.png">

```
<div xmlns="http://www.w3.org/1999/xhtml" id="console-msg-8403" class="message cm-s-mozilla" category="output" severity="log" filter="null">
  <span class="timestamp devtools-monospace">15:17:00.491 </span>
  <span class="indent" style="width: 0px;"></span>
  <span class="icon" title="Log"></span>
  <span class="message-body-wrapper message-body devtools-monospace">
    <span>
      <span class="class-Date">
        <a class="cm-variable" draggable="false" href="#">Date</a>
        <span class="cm-string-2">
          2016-02-12T23:17:00.498Z
        </span>
      </span>
    </span>
  </span>
</div>
```

**Comment by linclark** [comment]
I did a quick analysis of where the markup is coming from in the current implementation:
![markedup_date](https://cloud.githubusercontent.com/assets/105464/13023794/33d3bb74-d1a1-11e5-9442-a42409f0c8d7.png)

**Comment by linclark** [comment]
This was fixed via #9
